docs/user-guide/station-device-provisioning.md

```diff
@@ -137,9 +137,9 @@ If you don't want to use the LoRa Device Provisioning CLI, in the following sect
             [8, 125, 0],
             [7, 125, 0],
             [8, 500, 0],
-            [0, 0, 0],
-            [0, 0, 0],
-            [0, 0, 0],
+            [-1, 0, 0],
```

Comment:
As in the other PR, please update also AU915 example DR 7 to -1 

https://lora-alliance.org/wp-content/uploads/2020/11/lorawan_regional_parameters_v1.0.3reva_0.pdf

Reply:
Done with d69d64f9